### PR TITLE
no need for polling nupnp in the background

### PR DESCRIPTION
**Pull request by BenediktNiehues**
Signed-off-by: Benedikt Niehues <benedikt.niehues@itemis.de>

### CODE REVIEW

**Comment by maggu2810** [comment]
Any reason to override the `startBackgroundDiscovery` and `stopBackgroundDiscovery` at all? The implementation of the super class already "does nothing".

**Comment by BenediktNiehues** [comment]
@maggu2810 just wanted to leave the comments there
and as you can see, travis passed for this branch...

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java [hunk 1]
```diff
@@ -57,34 +55,20 @@
...
     private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = Collections.singleton(THING_TYPE_BRIDGE);
 
-    private static final int BRIDGE_SCAN_INTERVAL = 60;
-
     private final Logger logger = LoggerFactory.getLogger(HueBridgeNupnpDiscovery.class);
 
-    private ScheduledFuture<?> bridgeScanningJob;
-
     public HueBridgeNupnpDiscovery() {
         super(SUPPORTED_THING_TYPES, DISCOVERY_TIMEOUT);
```
Comment (kaikreuzer) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java]: you should actually call super with `false` as a third parameter.

**Comment by kaikreuzer** [comment]
I agree with @maggu2810 that the methods should not be there at all, if the service does not support background discovery.

File: extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java (hunk 1 above)
Comment (BenediktNiehues) [inline extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/internal/discovery/HueBridgeNupnpDiscovery.java]: good hint, thanks!

**Pull request merged**